utilities/game.py

```diff
@@ -123,8 +126,10 @@ def get_score_location(node: Node) -> tuple[Pose2d, Rotation2d]:
     return goal, approach
 
 
-def get_closest_node(pos: Translation2d, piece: GamePiece, row: Rows) -> Node:
+def get_closest_node(pos: Translation2d, piece: GamePiece, row: Rows, impossible: list[int]) -> Node:
```

Comment:
Since only membership is being checked in `impossible`
```suggestion
def get_closest_node(pos: Translation2d, piece: GamePiece, row: Rows, impossible: set[int]) -> Node:
```